elektronLogging/loggerConfig.go

```diff
@@ -0,0 +1,68 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	elekEnv "github.com/spdfg/elektron/environment"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+type LoggerConfig struct {
+	SchedTraceConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		AllowOnConsole      bool   `yaml:"allowOnConsole"`
+	} `yaml:"schedTrace"`
+
+	PCPConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		AllowOnConsole      bool   `yaml:"allowOnConsole"`
+	} `yaml:"pcp"`
+
+	ConsoleConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		MinLogLevel         string `yaml:"minLogLevel"`
+	} `yaml:"console"`
+
+	SPSConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		AllowOnConsole      bool   `yaml:"allowOnConsole"`
+	} `yaml:"sps"`
+
+	TaskDistConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		AllowOnConsole      bool   `yaml:"allowOnConsole"`
+	} `yaml:"clsfnTaskDistOverhead"`
+
+	SchedWindowConfig struct {
+		Enabled             bool   `yaml:"enabled"`
+		FilenameExtension   string `yaml:"filenameExtension"`
+		EnableColumnHeaders bool   `yaml:"enableColumnHeaders"`
+		AllowOnConsole      bool   `yaml:"allowOnConsole"`
+	} `yaml:"schedWindow"`
+
+	Format []string `yaml:"format"`
+}
+
+func (c *LoggerConfig) GetConfig() *LoggerConfig {
+
+	yamlFile, err := ioutil.ReadFile(elekEnv.LogConfigYaml)
+	if err != nil {
+		log.Printf("yamlFile.Get err   #%v ", err)
```

Comment:
The error message does not do justice in indicating the reason for the error.

Reply:
Changed the error message